scalyr_agent/log_processing.py

```diff
@@ -2272,6 +2273,7 @@ def perform_processing(self, add_events_request, current_time=None):
         """
         if current_time is None:
             current_time = time.time()
+        current_time_nano = int(current_time * 1000000000)
```

Comment:
Can we use ``get_nanoseconds_since_epoch`` from the new client library here? (will need to use a conditional import)

Reply:
If you're okay with another conditional import here then yes, I wanted to avoid it since for something this simple its very ugly but I'm not married to that idea.